### DISCUSSION TITLE
Is it possible to make all hrefs open in a new tab?

### DISCUSSION THREAD

**Discussion by aghaynes** [quarto-dev/quarto-web#354] [General]
It's possible to make a single link to another page with 

```
[some text](some-url){target="_blank"}
```
so that the page opens in a new tab. 

Is there a setting such that all urls that go outside of the quarto site use a blank target (perhaps through a header option), or does each link need the `{target}` attribute?

By the way, quarto is great! So powerful!

**Comment by jjallaire** [comment]
You could write a Lua filter that adds the `target` attribute to all of your links?

**Reply by aghaynes** [reply]
thanks for the hint :smile:
